.github/workflows/ci-weekly.yml

```diff
@@ -1,74 +1,164 @@
-name: CI - FAST Validation
+name: CI - FAST
 
 on:
-  schedule:
-  - cron: 0 7 * * 3
-
+  push:
+    branches:
+    - releases/*
+  pull_request:
+    branches:
+    - master
+    - releases/*
+    - features/*
 
 jobs:
-  build_windows:
-    name: Validate on Windows
-    runs-on: windows-latest
-
+  build_linux:
+    runs-on: [ubuntu-latest]
     steps:
     - uses: actions/checkout@v2
```

Comment:
For readability, please add new lines after all `runs-on: ...` and after each task within each step.  There are a few that are not consistent when viewing the final version that will be merged.

Reply:
okay